### ISSUE TITLE
Only owners should have permissions to eliminate collaborators from projects

### ISSUE THREAD

**Issue by jgwerner** [3Blades/3blades#175]
Right now any collaborator can add or delete other collaborators from Projects where the collaborator is not an Owner.

Only owners should be able to add/delete collaborators from projects.

**Comment by jgwerner** [comment]
@johngriebel once adding a new collaborator to a public project the project is duplicated with the same name.

**Comment by johngriebel** [comment]
@jgwerner the issue mentioned in your second comment has been resolved.

**Comment by johngriebel** [comment]
@jgwerner users should be able to remove _themselves_ as a collaborator on a project, right?

**Comment by jgwerner** [comment]
@johngriebel Yes, if possible. Perhaps you would we should open that as a separate feature request.
